circle.yml

```diff
@@ -22,5 +22,24 @@ deployment:
     branch: master
     commands:
       - docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS
-      - docker tag cyclus/cyclus:latest cyclus/cyclus:stable
-      - docker push cyclus/cyclus:stable
+      # Update cyclus/cylus:stable docker container
+      - docker tag cyclus/cyclus:latest cyclus/cyclus:stable # creation
+      - docker push cyclus/cyclus:stable # push to docker depot
+        ## DEB GENERATION
+          #ubuntu 14.04
```

Comment:
Does YAML support functions to reuse code?  The 14 and 16 code are identical except for the "14" or "16"